debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -509,6 +510,17 @@ public boolean isIncludePreImage() {
             .withImportance(Importance.LOW)
             .withDescription("The maximum processing time in milliseconds to wait for the oplog cursor to process a single poll request");
 
+    public static final Field CURSOR_PIPELINE = Field.create("cursor.pipeline")
+            .withDisplayName("Pipeline expression apply to the change stream cursor")
+            .withType(Type.STRING)
+            .withWidth(Width.SHORT)
+            .withImportance(Importance.LOW)
+            .withValidation(MongoDbConnectorConfig::validateAggregationPipeline)
+            .withDescription("Applies processing to change events as part of the the standard MongoDB aggregation stream pipeline. " +
+                    "A pipeline is a MongoDB aggregation pipeline composed of instructions to the database to filter or transform data. " +
+                    "This can be used customize the data that the connector consumes. " +
+                    "Note that this comes after the internal pipelines used to support the connector (e.g. filtering database and collection names).");
```

Comment:
Curious how this is used vs the AsciiDoc documents. Why have 2?

Reply:
This is used in by Debezium UI I believe.

Reply:
Yes, it is. All Field/s are. ;)

Reply:
Well that's the idea... they don't always much exactly as these are written mainly by engineering and "real" docs are often edited by documentation people.

Reply:
Just a comment. Please disregard.